include/swift/AST/DiagnosticsSema.def

```diff
@@ -2040,6 +2040,9 @@ ERROR(enumstruct_convenience_init,none,
 ERROR(nonclass_convenience_init,none,
       "convenience initializer not allowed in non-class type %0",
       (Type))
+ERROR(cfclass_convenience_init,none,
+      "convenience initializer not yet supported for extensions of CF Types",
```

Comment:
Also a plural / singular mismatch? "convenience initializers are not yet supported in extensions of CF types".